master/internal/experiment_filter.go

```diff
@@ -102,7 +102,7 @@ func expColumnNameToSQL(columnName string) (string, error) {
 		"duration":        "extract(epoch FROM coalesce(e.end_time, now()) - e.start_time)",
 		"state":           "e.state",
 		"numTrials":       "(SELECT COUNT(*) FROM trials t WHERE e.id = t.experiment_id)",
-		"progress":        "COALESCE(progress, 0)",
+		"progress":        "ROUND(COALESCE(progress, 0) * 100)::INTEGER", // multiply by 100 for percent
```

Comment:
used `ROUND` since our web code is 

https://github.com/determined-ai/determined/blob/ff830f96374d9a04f96d2792e4e66a33ad90d8c5/webui/react/src/utils/string.ts#L80

[toFixed](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) function works like round